features/pending_feedback_controller.feature

```diff
@@ -0,0 +1,66 @@
+Feature: get iteration feedback from customers who haven't submitted feedback yet
+	As a customer
```

Comment:
You could make this a bit more precise like you did for the iterations' feedback test and specify the user is logged in.